Scrips.BaseDbContext/Scrips.BaseDbContext/MaskValueAuditAttribute.cs

```diff
@@ -6,9 +6,9 @@
 
 namespace Scrips.BaseDbContext
 {
-    public class AuditIngore : Attribute
+    public class MaskValueAuditAttribute : Attribute
```

Comment:
Add the documentation found on confluence

Reply:
okay